spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql-r2dbc/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am confused about this starter -- is it for both, postgres and mysql at the same time?

Reply:
Sorry about that. Removing this sample. I must've missed it while resolving merge conflicts.